content/docs/gke/gcp-e2e.md

```diff
@@ -89,768 +89,29 @@ In summary:
 
 It's time to get started!
 
-## Set up your environment
+### Run the MNIST tutorial on GCP
 
-### Download the project files
+1. Follow the [GCP instructions](https://www.kubeflow.org/docs/gke/deploy/) to deploy Kubeflow with IAP
 
-To simplify this tutorial, you can use a set of prepared files that include
-a TensorFlow application for training your model, a web UI to send prediction
-requests and display the results, and the [Docker][docker] files to build
-runnable containers for the training and prediction applications.
-The project files are in the [Kubeflow examples
-repository](https://github.com/kubeflow/examples/tree/master/mnist)
-on GitHub.
+1. Launch a Jupyter notebook
 
-Clone the project files and go to the directory containing the MNIST example:
+   * The tutorial has been tested using the Jupyter Tensorflow 1.15 image
 
-```
-cd ${HOME}
-git clone https://github.com/kubeflow/examples.git
-cd examples/mnist
-WORKING_DIR=$(pwd)
-```
+1. Launch a terminal in Jupyter and clone the kubeflow examples repo
```

Comment:
s/kubeflow/Kubeflow

(Because it's a brand name, and we're not using it in a code snippet in this instance.)